Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -140,35 +133,30 @@ export RGW_POOL_PREFIX=default
     rook-ceph-external   /var/lib/rook                162m   Connected   HEALTH_OK
     ```
 
-6. StorageClass will also be created, verify its creation. `ceph-rbd` and `cephfs` StorageClass would be respective name for RBD and CephFS StorageClass.
+6. StorageClass will also be created depending on pools and fileysytem provided,
+verify its creation. `ceph-rbd` and `cephfs` StorageClass would be respective name for RBD and CephFS StorageClass.
     ```console
     kubectl -n rook-ceph-external get sc
     ```
 
 7. Then you can now create a [persistent volume](https://github.com/rook/rook/tree/master/deploy/examples/csi) based on these StorageClass.
 
-### CephCluster example (management)
-
-The following CephCluster CR represents a cluster that will perform management tasks on the external cluster.
-It will not only act as a consumer but will also allow the deployment of other CRDs such as CephFilesystem or CephObjectStore.
-You would need to inject the admin keyring for that.
-
-The corresponding YAML example:
-
-```yaml
-apiVersion: ceph.rook.io/v1
-kind: CephCluster
-metadata:
-  name: rook-ceph-external
-  namespace: rook-ceph-external
-spec:
-  external:
-    enable: true
-  dataDirHostPath: /var/lib/rook
-  cephVersion:
-    image: quay.io/ceph/ceph:v17.2.6 # Should match external cluster version
-```
+### Connect to an External Object Store
 
+1. Create [object-external.yaml](https://github.com/rook/rook/blob/master/deploy/examples/object-external.yaml), [storageclass-bucket-delete.yaml](https://github.com/rook/rook/blob/master/deploy/examples/storageclass-bucket-delete.yaml), [object-bucket-claim-delete.yaml](https://github.com/rook/rook/blob/master/deploy/examples/object-bucket-claim-delete.yaml).(Configure parmater names as per the CRD created)
```

Comment:
Provide a separate bullet for each one with its description so they know why to create them
```suggestion
Create the object store resources:
1. [External object store](https://github.com/rook/rook/blob/master/deploy/examples/object-external.yaml)[storageclass-bucket-delete.yaml]. (TELL WHY)
2. [Storage class for buckets](https://github.com/rook/rook/blob/master/deploy/examples/storageclass-bucket-delete.yaml). (TELL WHY)
3. [Object Bucket Claim](https://github.com/rook/rook/blob/master/deploy/examples/object-bucket-claim-delete.yaml). (TELL WHY)
```